src/renderer/map/Map.js

```diff
@@ -1,68 +1,42 @@
 import { ipcRenderer, remote } from 'electron';
 import fs from 'fs';
 import L from 'leaflet';
-import path from 'path';
 import React, { Component, createRef } from 'react';
 import { Map, Marker, Popup } from 'react-leaflet';
 
+import { cache, images, locations, startLocation } from '../../../resources/index.js';
 import CachedTileLayer from './CachedTileLayer.js';
 import GeolocationControl from './GeolocationControl.js';
 
-import 'leaflet/dist/leaflet.css';
 import './map.css';
 
-const vehicleIcons = {};
 const mapOptions = {
   attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
   url: 'https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}',
   id: 'mapbox.satellite',
-  accessToken: remote.getGlobal('process').env.MAPBOX_TOKEN,
-  useCache: false,
-  useOnlyCache: false,
+  accessToken: process.env.MAPBOX_TOKEN,
+  useCache: cache,
   crossOrigin: true,
 };
 
-for (const file of fs.readdirSync(path.resolve(__dirname, '../../../resources/images/markers/vehicles'))) {
-  const name = file.split('.').slice(0, -1).join('.');
-  vehicleIcons[name] = require(`../../../resources/images/markers/vehicles/${file}`);
-}
-
-function getStartLocation() {
-  const { startLocation, locations } = require('../../../resources/locations.json');
-  if (!startLocation || !locations[startLocation]) {
-    return { latitude: 0, longitude: 0 };
-  }
-  return locations[startLocation];
+let start = { latitude: 0, longitude: 0, zoom: 18 };
+if (startLocation && locations[startLocation]) {
```

Comment:
what is the purpose of this logic? and why can we not move this inside the constructor?

Reply:
Just made another commit that readded locations.json.

Just in that crazy case that locations.json is not there, the program will not crash. Also I cannot put this inside the class since the state variable depends on this variable, and is declared in the constructor (in a weird way through ES7)

I would leave that code there though

Reply:
If you remember from last year, we needed a default start location for the map. From the code, we just set it to 0,0. If there's a set startLocation and that startLocation is valid, then the map will start at that location instead.

Usually geolocation will come and it will pan at your location a few seconds later.